docs/Using-Fleet/Mobile-device-management.md

```diff
@@ -8,7 +8,13 @@ To use MDM features you have to connect Fleet to Apple Push Certificates Portal.
 
 ## macOS updates
 
-Fleet uses [Nudge](https://github.com/macadmins/nudge) to encourage the installation of macOS updates.
+Native macOS update reminders are available in Fleet Free. To trigger these reminders, run the ["Schedule an OS update" MDM command](https://developer.apple.com/documentation/devicemanagement/schedule_an_os_update).
+
+End user macOS update reminders (via [Nudge](https://github.com/macadmins/nudge)) are avaiable in Fleet Premium.
```

Comment:
"available"

Reply:
I'd put this section second after the Nudge section. 

Then there's no need to mention that nudge is available in premium and then repeat that immediately after.